### PR TITLE
update Animatronics to 2.0a

### PR DESCRIPTION
**Pull request by pouar**
Lance90 released an update to the animatronics mod recently
http://smf.cataclysmdda.com/index.php?topic=8258.0

He asked me to create a pull request for it

### CODE REVIEW

File: data/mods/Animatronics/animatronics.json [hunk 1]
```diff
@@ -1,552 +1,878 @@
...
-        "death_function":"BROKEN",
-        "special_attack":["NONE"],
-        "description":"A bipedal animatronic fox, nearly twice your size.  Its teeth and claws look quite sharp.  Though it is a robot, it stinks like death.",
-        "flags":["SEES", "GRABS", "HEARS", "BASHES", "ATTACKMON", "ELECTRONIC", "NO_BREATHE", "VIS50"]
+	"luminance":0,
+	"hp":100,
+	"special_freq":[0],
+	"death_function":"BROKEN",
+	"special_attack":[["PARROT", 0]],
+	"description":"A bipedal animatronic fox, nearly twice your size.  Its teeth and claws look quite sharp.  Its right had is replaced with a hook.  It looks heavily worn and degraded.",
```
Comment (KA101) [inline data/mods/Animatronics/animatronics.json]: hand

Might want to condense that?  "[...]size.  Its right paw has been replaced by a hook, but said hook, the claws on the left paw, and teeth all look quite sharp, despite its overall worn-down appearance."

Comment (pouar) [inline data/mods/Animatronics/animatronics.json]: That actually increases the string by 45 characters

Comment (KA101) [inline data/mods/Animatronics/animatronics.json]: OK, but at least fix the hand.  ;-)

File: data/mods/Animatronics/speech.json [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        "volume" : 40
+    },{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_bear",
+        "sound" : "\"SAVE THEM\"",
+        "volume" : 40
+	},{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_bear",
+        "sound" : "\"HELP THEM\"",
```
Comment (KA101) [inline data/mods/Animatronics/speech.json]: Might nail these as too much FNaF, I'm thinking.  Blob can't speak at all IMO, much less coherently.

Comment (Rivet-the-Zombie) [inline data/mods/Animatronics/speech.json]: I don't recall the animatronic critters in FNaF's talking, so there's that.

Comment (KA101) [inline data/mods/Animatronics/speech.json]: I read something about getting messages in the minigames along these lines.  *shrug*  Blob can trigger an existing voice-box, sure, but conscious speech not so much.

Comment (Rivet-the-Zombie) [inline data/mods/Animatronics/speech.json]: A creepy, static-y voice-box is much better than normal speech anyway.

**Comment by KA101** [comment]
Minor issue, these'll probably need a faction and to be friendly to one another, as currently they'll cheerfully ignore you to fight amongst themselves.

**Comment by Rivet-the-Zombie** [comment]
*Haunting moos.*

**Pull request merged**